### PR TITLE
Reduce the use of DCSA-Core in cases where it is not necessary

### PR DESCRIPTION
**Pull request by nt-as**
These commits will enable the removal/scope reduction of:

- `BaseService` (removal)
- `ExtendedBaseService` reduced to `findAllExtended`
- `BaseController` (removal)
- `ExtendedBaseController` reduced a `@GetMapping` for getting all entities

To keep things easier to review, I have excluded the changes to repositories for now as they can come in a separate PR.

### CODE REVIEW

File: src/main/java/org/dcsa/core/events/service/FacilityService.java [hunk 1]
```diff
@@ -8,10 +8,12 @@
 import java.util.UUID;
 
 public interface FacilityService extends ExtendedBaseService<Facility, UUID> {
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/service/FacilityService.java:10]: shouldn't the extends ExtendedBaseService be removed?

File: src/main/java/org/dcsa/core/events/service/TimestampDefinitionService.java [hunk 2]
```diff
@@ -3,9 +3,12 @@
 import org.dcsa.core.events.model.OperationsEvent;
 import org.dcsa.core.events.model.TimestampDefinition;
 import org.dcsa.core.service.ExtendedBaseService;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 public interface TimestampDefinitionService extends ExtendedBaseService<TimestampDefinition, String> {
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/service/TimestampDefinitionService.java:9]: shouldn't extends ExtendedBaseService be removed?

File: src/main/java/org/dcsa/core/events/service/VesselService.java [hunk 3]
```diff
@@ -8,5 +8,11 @@
 
 public interface VesselService extends ExtendedBaseService<Vessel, UUID> {
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/service/VesselService.java:9]: shouldn't extends ExtendedBaseService be removed

**Review by pimg — CHANGES_REQUESTED** [review]

File: src/main/java/org/dcsa/core/events/service/FacilityService.java (hunk 1 above)
Comment (nt-as) [reply src/main/java/org/dcsa/core/events/service/FacilityService.java:10]: No, UI Support (application supporting the UI) provides a facility controller that relies on the core features (filter + sorting).

File: src/main/java/org/dcsa/core/events/service/TimestampDefinitionService.java (hunk 2 above)
Comment (nt-as) [reply src/main/java/org/dcsa/core/events/service/TimestampDefinitionService.java:9]: No, UI Support (application supporting the UI) provides a timestamp definition controller that relies on the core features (filter + sorting).

File: src/main/java/org/dcsa/core/events/service/VesselService.java (hunk 3 above)
Comment (nt-as) [reply src/main/java/org/dcsa/core/events/service/VesselService.java:9]: No, UI Support (application supporting the UI) provides a vessel controller that relies on the core features (filter + sorting).

**Review by pimg — APPROVED** [review]

**Pull request merged**